File: codeup/100000578/D.py
#!/usr/bin/python3

days_in_month = [0, 31, 28, 31, 30, 31, 30, 31, 31, 30, 31, 30, 31]


class my_date:
    def __init__(self, y, m, d):
        self.y = y
        self.m = m
        self.d = d

    def next_day_print(self):
        if days_in_month[self.m] != self.d:
            self.d += 1
        else:
            self.d = 1
            if self.m != 12:
                self.m += 1
            else:
                self.m = 1
                self.y += 1

        print("{:04d}-{:02d}-{:02d}".format(self.y, self.m, self.d))


M = int(input())

for i in range(M):
    y, m, d = [int(x) for x in input().split()]
    md = my_date(y, m, d)
    md.next_day_print()

File: codeup/100000578/D.cpp
#include <iostream>
using namespace std;

int days_in_month[13] = {0, 31, 28, 31, 30, 31, 30, 31, 31, 30, 31, 30, 31};

class my_date {
 public:
  my_date(int y, int m, int d);
  void next_day_print();

 private:
  int _y;
  int _m;
  int _d;
};
my_date::my_date(int y, int m, int d) : _y(y), _m(m), _d(d) {}
void my_date::next_day_print() {
  if (days_in_month[_m] != _d) {
    _d++;
  } else {
    _d = 1;
    if (_m != 12) {
      _m++;
    } else {
      _m = 1;
      _y++;
    }
  }
  printf("%04d-%02d-%02d\n", _y, _m, _d);

  return;
}

int main() {
  int n, y, m, d;
  scanf("%d", &n);
  while (n--) {
    scanf("%d %d %d", &y, &m, &d);
    my_date md(y, m, d);
    md.next_day_print();
  }

  return 0;
}